src/main/webapp/courseLoader.js

```diff
@@ -394,12 +394,6 @@
     return response.json(); // Parses JSON response into native JavaScript objects and returns a Promise.
   }
 
-  function onLoad() {
```

Comment:
why is this being deleted?

Reply:
I am putting it on the main HTML, was having some issue with it being on the js file for some reason.